### PR TITLE
feat(cli): use local version when available

### PR DESCRIPTION
**Pull request by ianwremmel**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,6 +86,7 @@
   "dependencies": {
     "chalk": "^2.3.2",
     "debug": "^3.1.0",
+    "detect-installed": "^2.0.4",
```
Comment (squirly) [inline package.json]: I do not see where this is `require`d in this PR. Is it needed?

File: bin/clark [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+  const root = findRoot(process.cwd(), dir => {
+    const clarkPath = path.resolve(dir, 'node_modules', '.bin', 'clark');
+    // eslint-disable-next-line no-sync
+    return fs.existsSync(clarkPath);
+  });
+
+  debug('found locally installed clark');
+  const localPath = path.resolve(root, 'node_modules', '.bin', 'clark');
+  debug(`loading clark at ${localPath}`);
+  require(localPath);
```
Comment (squirly) [inline bin/clark]: Should this instead require the cli file? Otherwise I think it will keep looking upward?

File: bin/clark [hunk 3]
```diff
@@ -1,5 +1,37 @@
 #!/usr/bin/env node
+
```
Comment (kourge) [inline bin/clark]: ?

File: package.json (hunk 1 above)
Comment (ianwremmel) [reply package.json]: good catch, thanks!

File: bin/clark (hunk 2 above)
Comment (ianwremmel) [reply bin/clark]: ah, yea. i was testing locally against the version installed in `server`, so i didn't run into that. fixing it.

**Pull request merged**